web/src/features/moreCast2/pages/MoreCast2Page.tsx

```diff
@@ -134,11 +144,17 @@ const MoreCast2Page = () => {
   }, [stationPredictionsAsMoreCast2ForecastRows, selectedStations])
 
   useEffect(() => {
-    const predictions = fillInTheBlanks(fireCenter?.stations || [], stationPredictions, dateInterval, modelType)
+    const predictions = fillInTheModelBlanks(fireCenter?.stations || [], stationPredictions, dateInterval, modelType)
     const newRows = parseModelsForStationsHelper(predictions)
     setStationPredictionsAsMoreCast2ForecastRows(newRows)
   }, [stationPredictions]) // eslint-disable-line react-hooks/exhaustive-deps
 
+  useEffect(() => {
+    const completeDailies = fillInTheYesterdayDailyBlanks(fireCenter?.stations || [], yesterdayDailies, dateInterval)
```

Comment:
I think we need a check in here (or somewhere else) to see if the startDate is later than the endDate. In this scenario we shouldn't be showing any data, but we currently show data for a single day which is the day before the selected start date.

Reply:
good catch Darren!